protocol/messenger.go

```diff
@@ -2348,6 +2348,12 @@ func (m *Messenger) SyncDevices(ctx context.Context, ensName, photoPath string,
 			return err
 		}
 	}
+
+	err = m.syncAllKeycards(ctx, rawMessageHandler)
+	if err != nil {
```

Comment:
Hi @saledjenic , can you elebrate why only syncAllKeycards which corresponding protobuf.SyncAllKeycards within SyncDevices? we don't take care of protobuf.SyncKeycardAction within SyncDevices?

Reply:
@qfrank yes, sure... this is how UI looks in the desktop app:

`SyncKeycard` message is used to send details about each keycard change to enabled paired devices so they can update the their state for changed keycard.

But if two devices are using the same seed phrase and they are not paired, or pairing was disabled at the moment when some change were made on one of devices, user is able to sync the enabled paired devices clicking on `Sync All Devices` button (image below), in that case we're sending `SyncAllKeycards` message which contains the list of `SyncKeycard`s  that the paired devices can update their states based on that list. Cause in that case the device on which `Sync All Devices` button was clicked becomes source of truth.

![qq](https://user-images.githubusercontent.com/86303051/221168932-1d65d074-d9da-4421-b6fc-df12bfb82a1d.png)

Reply:
sorry, my question maybe not clear, i mean why we don't process syncing/dispatch protobuf.SyncKeycardAction within SyncDevices so that another device will receive message  which's type is protobuf.SyncKeycardAction? I knew syncAllKeycards will dispatch message protobuf.SyncKeycard and i have no question about it :) @saledjenic

Reply:
@qfrank we do that on each keycard action.

Please check the `dispatchKeycardActivity` function here https://github.com/status-im/status-go/blob/66373d4e0201796222026ce263948c0069be3807/protocol/messenger_keycard.go#L87-L113

Check this [video](https://drive.google.com/file/d/1FnOUlld0iSZPrDz-eGqyrbzfnlXguPrr/view?usp=share_link) you will see that each action is dispatched and changes are received and applied on other device.